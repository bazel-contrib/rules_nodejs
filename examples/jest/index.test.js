import indexfile from './index.js';

test('it should work', () => {
  expect(indexfile).toBe('test');
});

test('snapshot', function() {
  expect(indexfile).toMatchSnapshot();
})
