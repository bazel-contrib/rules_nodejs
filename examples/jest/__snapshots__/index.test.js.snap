// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `"test"`;
